src/index.js

```diff
@@ -1,17 +1,17 @@
+/* eslint-disable no-unused-vars */
+
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App.js';
+import Calculator from './components/Calculator.js';
 import reportWebVitals from './reportWebVitals.js';
```

Comment:
- [ ] Please remove all the boilerplate that was created that you will not use in your app like(
texts, images, and styles)